### PR TITLE
fix: fix minor issues of dashboard

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
please see order by commits!!!

1. [[Main Dashboard] 데이터 로딩 이슈](https://pyengine.atlassian.net/browse/QA-609)
2. [[Project Dashboard] 프로젝트별로 Daily Updates 위젯에 데이터가 없을때 표시가 다름](https://pyengine.atlassian.net/browse/QA-611)
3. received quest orally
4. [[Project Dashboard] 프로젝트별로 Billing 위젯의 [Day], [Month] 탭 정상 작동여부가 상이함](https://pyengine.atlassian.net/browse/QA-610)

### CODE REVIEW

File: apps/web/src/common/modules/widgets/DailyUpdates.vue [hunk 1]
```diff
@@ -14,33 +14,31 @@
...
                         />
+                        <div class="grid grid-cols-1 gap-1 w-full">
+                            <p-skeleton width="80%"
+                                        height="0.625rem"
+                            />
+                            <p-skeleton width="100%"
+                                        height="0.625rem"
+                            />
+                        </div>
                     </div>
```
Comment (WANZARGEN) [inline apps/web/src/common/modules/widgets/DailyUpdates.vue:40]: suggestion: You can give parent <div> to prevent customizing css with deep. :)

**Review by WANZARGEN — APPROVED** [review]
LGTM~ I left one suggestion!

File: apps/web/src/common/modules/widgets/DailyUpdates.vue [hunk 2]
```diff
@@ -430,7 +413,17 @@ export default {
...
+:deep(.p-data-loader) {
+    .data-loader-container {
+        .loader-wrapper {
+            .loader {
+                display: block;
+            }
+        }
+    }
+}
+.p-data-loader > .data-loader-container > .loader-wrapper > .loader .card-wrapper .daily-update-card-alert {
```
Comment (skdud4659) [inline apps/web/src/common/modules/widgets/DailyUpdates.vue:426]: question: what's the difference with line 417?

**Review by skdud4659 — APPROVED** [review]
LGTM!

File: apps/web/src/common/modules/widgets/DailyUpdates.vue (hunk 1 above)
Comment (yuda110) [reply apps/web/src/common/modules/widgets/DailyUpdates.vue:40]: Due to time constraints, I will refactor it next time!🥲

File: apps/web/src/common/modules/widgets/DailyUpdates.vue (hunk 2 above)
Comment (yuda110) [reply apps/web/src/common/modules/widgets/DailyUpdates.vue:426]: Oh, my mistake. Thanks!!! 😂

**Pull request merged**